util/dvsim/LintCfg.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""
+Class describing simulation configuration object
+"""
+
+import logging as log
+import sys
+from pathlib import Path
+
+from tabulate import tabulate
+
+from .BuildOnlyCfg import BuildOnlyCfg
+from .Deploy import *
+from .Modes import *
+from .utils import *
+
+
+# helper function for printing messages
+def _print_msg_list(msg_list_name, msg_list):
+    md_results = ""
+    if msg_list:
+        md_results += "### %s\n" % msg_list_name
+        md_results += "```\n"
+        for msg in msg_list:
+            msg_parts = msg.split()
+            md_results += msg + "\n\n"
+        md_results += "```\n"
+    return md_results
+
+
+# conditionally add markup to an html cell
+def _print_cell(entry, is_warning=False, no_markup=False):
+    if isinstance(entry, str) or no_markup:
+        cell_str = '<td align="center">'
+    elif entry == 0:
+        cell_str = '<td align="center" class="c10">'
+    elif is_warning:
+        cell_str = '<td align="center" class="c5">'
+    else:
+        cell_str = '<td align="center" class="c0">'
+    return cell_str + str(entry) + '</td>\n'
+
+
+class LintCfg(BuildOnlyCfg):
+    """Derivative class for linting purposes.
+    """
+    def __init__(self, flow_cfg_file, proj_root, args):
+        super().__init__(flow_cfg_file, proj_root, args)
+
+    @staticmethod
+    def create_instance(flow_cfg_file, proj_root, args):
+        '''Create a new instance of this class as with given parameters.
+        '''
+        return LintCfg(flow_cfg_file, proj_root, args)
+
+    def gen_results_summary(self):
+        # '''
+        # Gathers the aggregated results from all sub configs
+        # '''
```

Comment:
If this is intended to be a doc comment remove the `#`

Reply:
Indeed, fixed.